Makefile

```diff
@@ -5,20 +5,20 @@ GO=go
 DIST_ROOT=dist
 # We specify version for the build; it is the branch-name by default, also we try
 # to find if there is a tag pointed to the current commit. If so, we use the tag.
-DIST_VER=$(shell git rev-parse --abbrev-ref HEAD)
-ifeq ($(shell git describe --tags $(git rev-parse @) >&/dev/null; echo $$?), 0)
-	DIST_VER=$(shell git describe --tags $(git rev-parse @))
+DIST_VER=$(git rev-parse --abbrev-ref HEAD)
```

Comment:
Interestingly, after we remove `shell` from all of these lines they stop working for me. Using zsh/mac, using GNU Make 3.81

Reply:
Okay, need to replace the `>&` then I guess.